src/Components/test/testassets/BasicTestApp/wwwroot/index.html

```diff
@@ -30,6 +30,11 @@
             // Load either blazor.webassembly.js or blazor.server.js depending
             // on the hash part of the URL. This is just to give a way for the
             // test runner to make the selection.
+            if (location.hash === '#server') {
+                location.hash = '';
+                document.cookie = '__blazor_execution_mode=server';
```

Comment:
Does this break other tests that use cookies?

Reply:
Nope, that's why I picked the cookie name